src/backend/storeManagers/gog/user.ts

```diff
@@ -115,26 +115,6 @@ export class GOGUser {
     return JSON.parse(stdout)
   }
 
-  /**
-   * Migrates existing authorization config to one supported by gogdl
```

Comment:
I checked with linguin, we don't need this anymore, it was here temporarily to migrate credentials over a few releases